services/mining/newblocktemplate.go

```diff
@@ -492,13 +444,7 @@ mempoolLoop:
 		Blues:           blues,
 		ValidPayAddress: payToAddress != nil,
 		PowDiffData: types.PowDiffStandard{
```

Comment:
If `PowDiffStandard` is some kind of redundancy to contain a compact diff only, should `PowDiffData` exist anymore if it takes same value with `Difficulty`?